Web/Cube/src/WebARCube.xml

```diff
@@ -0,0 +1,308 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<widget
+    id="com.mendix.widget.web.webarcube.WebARCube"
```

Comment:
Hey, almost all widgets are missing the `<helpUrl>` tag in xml's, is this a known issue?

Reply:
I didn't know about it, but I've added it to all widgets now.